docs/source/04_user_guide/08_advanced_io.md

```diff
@@ -84,6 +88,8 @@ my_dataset:
   type: <path-to-my-own-dataset>.MyOwnDataSet
   filepath: data/01_raw/my_data.csv
   versioned: true
+  param1: <param1-value>
+  param2: <param2-value>
```

Comment:
Maybe make one parameter optional and one mandatory, for completeness?

Reply:
param2 is now `True` by default